test/com/xilinx/rapidwright/design/TestSiteRoutingResolver.java

```diff
@@ -0,0 +1,55 @@
+package com.xilinx.rapidwright.design;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.xilinx.rapidwright.device.BELPin;
+import com.xilinx.rapidwright.support.CheckOpenFiles;
+import com.xilinx.rapidwright.support.RapidWrightDCP;
+import com.xilinx.rapidwright.tests.CodePerfTracker;
+import com.xilinx.rapidwright.util.Pair;
+
+public class TestSiteRoutingResolver {
```

Comment:
Should we stick this inside `TestDesignTools.java` (it might be a good idea to follow the same structure as `:/src`) which could eventually end up as a big monolithic file, but does not impact the ability to run specific tests, or should we have a `DesignTools` subdir where this can sit?

Reply:
I thought about that, but in the past it was mentioned that we should break up the tests to facilitate parallelization.

Reply:
I think there's still opportunity for parallelization at the class granularity given I expect that (eventually) there will be many classes.